owncloudApp/src/main/java/com/owncloud/android/presentation/UIResult.kt

```diff
@@ -41,6 +43,15 @@ sealed class UIResult<out T> {
             is Error -> error
             else -> null
         }
+
+    companion object {
+        fun <T> fromResult(result: UseCaseResult<T>): UIResult<T> =
+            if (result.isSuccess) {
+                Success(result.getDataOrNull())
+            } else {
+                Error(result.getThrowableOrNull())
+            }
+    }
```

Comment:
Why do we have a companion function and an extension that do exactly the same? I think that they could be merged into one. What do you think?

Reply:
I just removed it.